src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js

```diff
@@ -1,7 +1,7 @@
 import { useImmerReducer } from "use-immer";
 
 const initialNewAnnotationState = {
-	newAnnotation: {},
+	newAnnotation: {"negated": false},
```

Comment:
Does this field need to be specified here?

Reply:
I think so, otherwise it gets cleared when you do CLEAR

Reply:
We were trying to not set any fields in the object, and just handle any issues elsewhere.

Reply:
You could just default it in the clear call.

Reply:
I tried just setting it in the 'CLEAR' case (as it was previously done in the 'OPEN_DIALOG' case) but it was throwing errors the second time you hit clear: 
`Uncaught TypeError: Cannot assign to read only property 'negated' of object '#<Object>'`

Reply:
So doing a `draft.newAnnotation["negated"] = false;` in the CLEAR was throwing that error?

Reply:
yep, but only on the 2nd time you hit Clear

Reply:
and only if you'd entered something in one of the non-list fields